main.tf

```diff
@@ -221,8 +221,3 @@ resource "lacework_integration_aws_ct" "default" {
 
   depends_on = [time_sleep.wait_time]
 }
-
-data "lacework_metric_module" "lwmetrics" {
```

Comment:
You could make it a conditional data source
```
  count = var.enable_metric_module ? 1 : 0
```

Reply:
sure, updated.